sql/2020/18_Page_Weight/18_01.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 18_01: Distribution of page weight by resource type and client.
```

Comment:
Please update the query file names to be more descriptive of the metrics themselves. For example `bytes_per_type_2020.sql`